src/main/java/org/overture/ego/model/entity/UserPermission.java

```diff
@@ -40,12 +40,12 @@
   UUID id;
 
   @NonNull
-  @ManyToOne(fetch = FetchType.LAZY)
+  @ManyToOne(fetch = FetchType.EAGER)
```

Comment:
why?

Reply:
This change fixes a bug with JSON serialization.